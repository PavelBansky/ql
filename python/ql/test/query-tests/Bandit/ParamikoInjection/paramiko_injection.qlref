Bandit/paramiko_injection.ql