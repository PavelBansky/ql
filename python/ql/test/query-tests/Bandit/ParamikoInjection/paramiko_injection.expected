| Test.py:7:1:7:48 | Attribute() | Possible shell injection via Paramiko call, check inputs are properly sanitized. |
